### PR TITLE
Make compatiable with ruby 3

### PR DESCRIPTION
**Pull request by aniketstiwari**

### CODE REVIEW

**Comment by aniketstiwari** [comment]
@kyleboe  When you get a chance. Could you please review it?

File: CHANGELOG.md [hunk 1]
```diff
@@ -86,4 +86,4 @@ This is the first CHANGELOG entry.
 ### Misc:
 * Update gems
 * Update bundler to 2.2
-* Update Ruby to 2.7
+* Update Ruby to 3.0
```
Comment (kyleboe) [inline CHANGELOG.md:89]:
```suggestion
* Update Ruby to 2.7
```
Lets revert this change. I'll add an entry to update the changelog after we get this merged.

Comment (aniketacima) [reply CHANGELOG.md]: @kyleboe  Done

Comment (aniketstiwari) [reply CHANGELOG.md]: @kyleboe  Done

**Review by kyleboe — APPROVED** [review]

**Comment by kyleboe** [comment]
Thanks for the update! I'll get a new version of the gem published soon.

**Pull request merged**